tests/integration/test_focal.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
I think this file could be dropped and you could use pytest's parameterize feature in the other integration test to just deploy for both series'

Reply:
Great suggestion Jon. I changed the code to use `pytest's parametrize` and also moved the charm `build` call to a `fixture` in order to run it just once (as we don't need different builds at this time for the two different series).

Reply:
I also moved the expected content of the configuration files from the code to files in the repo in order to not duplicate them, as they are used in multiple source and test files.